obal/data/modules/srpm.py

```diff
@@ -0,0 +1,122 @@
+"""
+Build SRPM
+"""
+
+import shutil
+import os
+import subprocess
+from tempfile import mkdtemp
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.chdir import chdir # pylint:disable=import-error,no-name-in-module
+
+def run_command(command):
+    """
+    Run a system command
+    """
+    return subprocess.check_output(
+        ' '.join(command),
+        universal_newlines=True,
+        shell=True,
+        stderr=subprocess.STDOUT
+    )
+
+
+def copy_sources(spec_file, package_dir, sources_dir):
+    """
+    Copy RPM sources to rpmbuild environment
+    """
+    command = ["spectool", "--list-files", spec_file]
+    sources = run_command(command)
+
+    for source in sources.split("\n"):
+        if not source:
+            continue
+
+        source_link = source.split(' ')[1]
+
+        if not source_link.startswith('http'):
+            shutil.copy(os.path.join(package_dir, source_link), sources_dir)
+
+    with chdir(package_dir):
+        run_command(["git-annex", "lock"])
+        annexed_files = run_command(["git-annex", "find", "--include='*'"])
+        annexed_files = annexed_files.splitlines()
+
+        if not annexed_files:
+            return
+
+        run_command(["git-annex", "get"])
+
+        try:
+            run_command(["git-annex", "unlock"])
+        finally:
+            for annex in annexed_files:
+                shutil.copy(annex, sources_dir)
+
+            run_command(["git-annex", "lock", "--force"])
+
+
+def main():
+    """
+    Build a package using tito
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            package=dict(type='str', required=False),
```

Comment:
technically that's the path to the package source directory, right?

Reply:
Yes it is.